internal/project/framework/internals/binder/bind.go

```diff
@@ -9,8 +9,9 @@ import (
 	"net/http"
 	"strings"
 
-	"{{ .PkgPath }}/framework/internals/helpers"
-	structure "{{ .PkgPath }}/framework/internals/struct"
+	/**#bean*/
+	"demo/framework/internals/helpers" /*#bean.replace("{{ .PkgPath }}/framework/internals/helpers")**/
```

Comment:
put these two lines in one line

Reply:
`go fmt` command make it, `go fmt` will run before I commit,but I don't know the fmt rules
bean macro still works in two lines